### ISSUE TITLE
Prevents child element from scrolling

### ISSUE THREAD

**Issue by joloco** [timbartsch/no-bounce#3]
Hi Tim, 

Thanks for producing no-bounce, it works very well for me!

Apart from... I have a pop-over element (a list, displayed by jQuery UI) whose contents are longer than the window. Without no-bounce, it works fine, users can scroll the contents, and the bounce doesn't affect the main window.

But using no-bounce, it prevents the contents of the UI box from scrolling. 

So, my question is – can no-bounce be restricted to just the main body element?

Thanks in advance!

**Comment by timbartsch** [comment]
Hi Joloco,

thanks for the feedback =)

Can you tell me the exact jQuery UI element you are using?
Or can you provide a test case? So that I can reproduce the issue.

Adding a function to exclude DOM elements from noBounce should be possible to implement.

**Comment by arti040** [comment]
Yea, it would be cool to have possibility to restrict preventing default behaviour for touch-move event to particular elements only. As @joloco said - now it's possible to scroll body element only but not any other, with overflow: scroll property defined, i.e. some DIV with content inside.

**Comment by timbartsch** [comment]
Hi arti040 now you can enable no-bounce for a single dom element of your choice.

Check out the new example and let me know if that fits your needs:
http://timbartsch.github.io/examples/no-bounce/

**Comment by ghost** [comment]
Thanks. I'll check it tomorrow and let you know. Have no access to iOS device now.

**Comment by ghost** [comment]
No, no. It's not what I meant. Now you still scroll content in BODY element, but event is registered to DIV. This, what I want, is to have possibility to scroll a content inside a particular DOM element, i.e. DIV with overflow:scroll, not just BODY. So, we need as follows:
- additional parameter, i.e. settings.element;
- if element is BODY, use document.scrollBy(x,y);
- if element is a DOM element, then use some custom function like scrollElementBy(x,y);

Is it now clear?

**Comment by timbartsch** [comment]
Do you want to prevent the bounce effect on scrolling in a DIV with overflow:scroll like here:

http://www.quirksmode.org/css/css2/overflowscrolling.html

**Comment by ghost** [comment]
This, what I want is a scrolling DIV (like in a link you provided) with bouncing effect disabled for _whole page_ - for that DIV as well as for a BODY. I just want to get rid of this stupid bouncing effect at all ;-)

**Comment by timbartsch** [comment]
OK now I got it.
Do you have a code example I could use?

**Comment by ghost** [comment]
I'm working on it now.

**Comment by batmanbury** [comment]
Hi @arti040 did this ever get anything work done on it? This plugin seems like my best hope for preventing bounce on specific elements, but when I use the element-specific approach, it prevents scrolling altogether

```
window.onload = function() {
  noBounce.init({
    animate: true,
    element: document.getElementById("content")
  });
}
```
